curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -62,18 +62,18 @@ fiboEvenSum(10);
 
 ```js
 const fiboEvenSum = (number) => {
-  let temp, sum = 0, a = 0, b = 1;
-    while (number >= 0) {
-      temp = a;
-      a = b;
-      b += temp;
-      number --;
-      if ((b % 2) === 0) {
-        sum += b;
+  if (number <= 1) {
+    return 0;
+  } else {
+      let evenSum = 2, first = 1, second = 2, fibNum; // According to problem description our Fibonacci series starts with 1, 2
+      for (let i = 3; i <= number; i++) {
+        fibNum = first + second;
+        first = second;
+        second = fibNum;
+        if (fibNum % 2 == 0) evenSum += fibNum;
```

Comment:
Small nitipck: We use braces even with single line if statements.

Reply:
@ezioda004 Done!